tests/test_utilities.py

```diff
@@ -156,6 +156,159 @@ def test_check_all_contiguous():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_interleaveImages():
+    import time
+    t1 = time.time()
+
+    # 1a) With galsim Gaussian
+    g = galsim.Gaussian(sigma=3.7,flux=1000.)
+    gal = galsim.Convolve([g,galsim.Pixel(1.0)])
+    im_list = []
+    offset_list = []
+    n = 2
+    for j in xrange(n):
+        for i in xrange(n):
+            im = galsim.Image(16*n,16*n)
+            offset = galsim.PositionD(-(i+0.5)/n+0.5,-(j+0.5)/n+0.5)
+            offset_list.append(offset)
+            gal.drawImage(image=im,scale=1.0,method='no_pixel',offset=offset)
```

Comment:
Don't use 1.0 for the scale here.  It's too easy for bugs to be missed when using 1.0 as a value in a test suite.